### PR TITLE
[DOC-10012] Improve docs about statement bundle privileges

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-10012] Improve docs about statement bundle privileges

### CODE REVIEW

File: src/current/v23.1/explain-analyze.md [hunk 1]
```diff
@@ -33,7 +33,16 @@ Parameter          | Description
 
 ## Required privileges
 
-The user requires the appropriate [privileges]({% link {{ page.version.version }}/security-reference/authorization.md %}#managing-privileges) for the statement being explained.
+To generate a statement bundle, you must have the [privileges]({% link {{ page.version.version }}/security-reference/authorization.md %}#managing-privileges) required by the statement. The user requires the appropriate [privileges]({% link {{ page.version.version }}/security-reference/authorization.md %}#managing-privileges) for the SQL statement being explained.
```
Comment (jonstjohn) [inline src/current/v23.1/explain-analyze.md:36]: I think the following might be slightly clearer:

"
To generate a statement bundle, you must have the privileges required to execute the SQL statement, as well as the privileges required to collect the statement bundle.
"

There are really 2 sets of privileges required when capturing the statement bundle using `EXPLAIN (ANALYZE)`.

**Comment by jonstjohn** [comment]
Other than the one comment - LGTM!

File: src/current/v23.1/explain-analyze.md (hunk 1 above)
Comment (mdlinville) [reply src/current/v23.1/explain-analyze.md]: Done, thank you!

**Review by rmloveland — APPROVED** [review]
LGTM!

**Pull request merged**